packages/nextcloud/test/webdav_test.dart

```diff
@@ -653,6 +653,26 @@ void main() {
 
         // props: Most of them are either not applicable or hard/impossible to implement because we don't allow just writing any props
       });
+
+      test('Upload and download empty file', () async {
+        final destinationDir = Directory.systemTemp.createTempSync();
+        final destination = File('${destinationDir.path}/empty-file');
+        final source = File('${destinationDir.path}/empty-file-source')..createSync();
+
+        await client.webdav.putFile(
+          source,
+          source.statSync(),
+          PathUri.parse('empty-file'),
+        );
+        await client.webdav.getFile(
+          PathUri.parse('empty-file'),
+          destination,
+          onProgress: (progress) => expect(progress, equals(1)),
```

Comment:
This test case needs to be changed as it only checks that the progress equals `1` if called and does not check that the progress is called at all.